src/compiler/checker.ts

```diff
@@ -20964,6 +20985,25 @@ namespace ts {
                         flags |= ModifierFlags.Async;
                         lastAsync = modifier;
                         break;
+
+                case SyntaxKind.OverrideKeyword:

```

Comment:
Also add a check in `checkAccessorDeclaration` for accessors matching in `override`